app/views/homepage/_newest_workshops.html.erb

```diff
@@ -0,0 +1,16 @@
+<% new_workshops = Workshop.all.order(created_at: :desc).limit(5) %>
+
+<% if new_workshops.any? %>
+  <h2>Newly Registered Workshops</h2>
+  <div class="row">
+    <ul class="list">
+      <% new_workshops.each do |workshop| %>
+        <li>
```

Comment:
**Change Request:** Use the existing (currently hidden) means of displaying a workshop on the homepage.

https://github.com/JiggyPete/global-diversity-cfp-day-site/blob/master/app/views/homepage/_workshops.html.erb#L11

This partial ☝️ displays all the workshops by continent and country and the block highlighted displays an individual workshop.

If we can extract that block to a new partial template file in `app/views/homepage/_workshop.html.erb` you can use it in your commit. The only other thing to do is pass in the `workshop: workshop` parameter from `_workshops.html.erb` and from your commit, then it'll be consistent when we get to launch.

![image](https://user-images.githubusercontent.com/67151/64472792-8c71ad00-d15b-11e9-9d21-ce8749983763.png)

Reply:
Hey @JiggyPete, I think I understand. I've updated the PR. Let me know if I'm way off your thoughts.